aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -117,6 +117,52 @@ Tensor pinverse(const Tensor& self, double rcond) {
   return V.mm(S_pseudoinv.diag().mm(U.t()));
 }
 
+static double _get_epsilon(const ScalarType& sc_type) {
```

Comment:
It would be nice to have this in ATen / PyTorch as `finfo` like NumPy.

Reply:
Can we use [c++ numeric_limits](https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon) instead?

Reply:
C++ numeric limits doesn't provide epsilon for `at::Half`. This value is there in `ATen/core/Half-inl.h', but I am not sure how to extract it.